packages/react-aria-components/docs/Calendar.mdx

```diff
@@ -422,7 +422,7 @@ The error message element within a `Calendar` can be targeted with the `[slot=er
 Multiple `CalendarGrid` elements can be rendered to show multiple months at once. The `visibleDuration` prop should be provided to the `Calendar` component to specify how many months are visible, and each `CalendarGrid` accepts an `offset` prop to specify its starting date relative to the first visible date.
 
 ```tsx example
-<Calendar aria-label="Appointment date" visibleDuration={{months: 2}}>
+<Calendar aria-label="Appointment date" visibleDuration={{months: 3}}>
```

Comment:
does this actually show 3 months? I'm only seeing two, I think we need to include a calendargrid down below with a month offset -1?

Reply:
Ahh good catch!

Reply:
Our other calendars have the center month as the current one, and then -1 and +1 on either side? or am I interpreting that differently?

Reply:
I don't think we do that anywhere. Not sure what the difference would be, but I'm happy to update that if its more accurate.

Reply:
ah, yep, i was mistaken, i thought they were centered, they are not